examples/examples_C++/convection_diffusion.cpp

```diff
@@ -0,0 +1,98 @@
+#include <iostream>
```

Comment:
Hi @SavannaSpayd  thank you so much for considering contributing to this community project. Any contributions are welcome! However, you seem to have submitted code that does not use any functionality of the MOLE library. That is not the intent of adding an example in the examples directory. We want examples that show the functionality of the library, so you need to use the MOLE built-in functions in your code.

You could still use this code for comparison purposes on your local machine, to verify the correctness of your MOLE example. But please do not submit this code to the PR. 

Also you seem to have checked out your branch from an old version of `master`. The directory tree has now changed on `master`, so now the new path for C++ examples is `examples/cpp`.

There are two ways to update your branch with the current master branch.

1. The [git merge](https://git-scm.com/docs/git-merge) way: Update the master branch by doing git checkout master and then git pull. Once master is up to date, you can merge the master branch into yours by running git checkout your_branch (i.e., go back to your own branch) and then git merge master. This way will merge all commits there are currently on master into your own feature branch. This way causes the commit history of branch to be mingled with your own local feature branch commit history. For this reason, I tend to avoid this way.

2. The [git rebase](https://git-scm.com/docs/git-rebase) way. Update the master branch by doing git checkout master and then git pull. Once master is up to date, you can rebase your branch on top of the master branch by running git checkout your_branch (i.e., go back to your own branch) and then git rebase master. This way your feature branch commit history will always be on top of the master branch commit history. It is a lot cleaner and you can easily identify your own commits via git log and change commit history if needed.


I hope this helps!